src/applications/login/routes.jsx

```diff
@@ -1,10 +1,17 @@
 import SignInApp from './containers/SignInApp';
 import SignInWrapper from './components/SignInWrapper';
+import MockAuth from './containers/MockAuth';
 
 const routes = {
   path: '/',
   component: SignInWrapper,
   indexRoute: { component: SignInApp },
+  childRoutes: [
+    {
+      path: 'mocked-auth',
+      component: MockAuth,
+    },
+  ],
```

Comment:
Let's have `childRoutes` conditionally be created for dev & localhost environments.